src/templates/actions-page.js

```diff
@@ -1,20 +1,35 @@
 import React from "react";
+import { useQuery } from "@apollo/react-hooks";
 
 import Layout from "../components/Layout";
+import Header from "../components/Header";
 import Campaign from "../sections/CampaignActions";
+import { GET_USER } from "../api";
 
-// TODO: this data needs to be pulled from our SSO service
-const tempFakeUser = {
-  id: "3",
-  name: "Jane Doe",
-  email: "jane.doe@mail.com"
-};
+// TODO: this campaignId needs to be pull from the server (mean while take from userActions table vs currentUser id)
```

Comment:
@duranmla Actually is fine if we hard code this as an env variable. 

We should add seeds in Knex to create the default campaign

Reply:
Let's create an issue within this project to address this one.
nevertheless, with the `UserCampaign` we can pull the campaign from there too